lib/rules/accessor-pairs.js

```diff
@@ -63,3 +63,11 @@ module.exports = function(context) {
     };
 
 };
+
+module.exports.documentation = {
```

Comment:
I like `meta`. Should `schema` be moved under `meta.schema`? (I haven't thought about a backwards-compatible way to do that yet, so it may not be possible.)